alliance.md

```diff
@@ -4,22 +4,21 @@ css_id: alliance
 ---
 
 
-# The Uptane Alliance
+# Uptane: A JDF/Linux Foundation Project
 
 The Uptane Alliance was formally established in July of 2018 for the express
 purpose of standardizing an open and secure design for over-the-air updates
 for automobiles.  In practice, this process has involved improving and
 standardizing the Uptane open-source software. Formed under the umbrella of
-IEEE's [International Standards and Technology Organization](https://ieee-isto.org/) and now 
-a [Linux Foundation Joint Development Foundation](http://www.jointdevelopment.org/) alliance, the
-Alliance is a nonprofit consortium of companies, non-profit groups, and government agencies.
+IEEE's [International Standards and Technology Organization](https://ieee-isto.org/), the legal name of the initiative is now the Joint Development Foundation
```

Comment:
Sorry, I'm confused. It was formed under IEEE/ISTO, but now only the legal name has changed, or is it now officially under JDF/LF?

Reply:
It is officially under JDF/LF. Let me clarify the wording.

Reply:
Done